src/main.tsx

```diff
@@ -16,13 +22,23 @@ ReactDOM.createRoot(document.getElementById("root")!).render(
       <BrowserRouter>
         <Routes>
           <Route path="/" element={<div>Home Route</div>} />
-          <Route path="/registration-confirm" element={<RegistrationConfirmation />} />
-          <Route path="/onboarding/glucose-notifications" element={<GlucoseNotificationPrompt />} />
+          <Route
+            path="/registration-confirm"
+            element={<RegistrationConfirmation />}
+          />
+          <Route
+            path="/onboarding/glucose-notifications"
+            element={<GlucoseNotificationPrompt />}
+          />
+          <Route
+            path="/onboarding/glucose-notifications-list"
+            element={<GlucoseNotificationList />}
```

Comment:
Small issue here, the ticket mentions this to be a component - evident by the tsx file missing the logo and the back button, so there is no need to route it here in main.